src/Provider/3dTilesProvider.js

```diff
@@ -203,7 +203,16 @@ function executeCommand(command) {
     const metadata = command.metadata;
     const tile = new THREE.Object3D();
     configureTile(tile, layer, metadata, command.requester);
-    const path = metadata.content ? metadata.content.url : undefined;
+    // Patch for supporting 3D Tiles pre 1.0 (metadata.content.url) and 1.0
+    // (metadata.content.uri)
+    let path;
```

Comment:
You can put a one liner: `const path = metadata.content && (metadata.content.url || metadata.content.uri);`. No need for a big `if` structure if the comment is explicit enough.

Reply:
Yep, you're right. It's done.